Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -17,30 +21,115 @@ public class SettingsMenu : PlatformDependentComponent
         /// </summary>
         private string SettingsPrefab => UIPrefabFolder + "SettingsMenu";
 
+        /// <summary>
+        /// Prefab for the KeyBindingContent.
+        /// </summary>
+        private string KeyBindingContent => UIPrefabFolder + "KeyBindingContent";
+
+        /// <summary>
+        /// Prefab for the ScrollView.
+        /// </summary>
+        private string ScrollPrefab => UIPrefabFolder + "ScrollPrefab";
+
         /// <summary>
         /// The game object instantiated for the <see cref="SettingsPrefab"/>.
         /// </summary>
         private GameObject settingsMenuGameObject;
 
         /// <summary>
-        /// Sets the text of the textbox and adds the onClick event ExitGame to the ExitButton.
+        /// A mapping of the short name of key binding onto the label of the button that allows to
+        /// change the binding. This dictionary is used to update the label if the key binding
+        /// was changed by the user.
+        /// </summary>
+        Dictionary<string, TextMeshProUGUI> shortNameOfBindingToLabel;
+
+        /// <summary>
+        /// Sets the <see cref="KeyBindingContent"/> and adds the onClick event
+        /// <see cref="ExitGame"/> to the ExitButton.
         /// </summary>
         protected override void StartDesktop()
         {
             // instantiates the SettingsMenu
             settingsMenuGameObject = PrefabInstantiator.InstantiatePrefab(SettingsPrefab, Canvas.transform, false);
-            Button exitButton = settingsMenuGameObject.transform.Find("SettingsPanel/ExitButton").gameObject.MustGetComponent<Button>();
-            // adds the ExitGame method to the button
-            exitButton.onClick.AddListener(ExitGame);
-            // sets the text of the scrollview
-            settingsMenuGameObject.transform.Find("KeybindingsPanel/KeybindingsText/Viewport/Content").gameObject.MustGetComponent<TextMeshProUGUI>().text = KeyBindings.GetBindingsText();
+            // adds the ExitGame method to the exit button
+            settingsMenuGameObject.transform.Find("ExitPanel/Buttons/Content/Exit").gameObject.MustGetComponent<Button>()
+                .onClick.AddListener(ExitGame);
+
+            shortNameOfBindingToLabel = new Dictionary<string, TextMeshProUGUI>();
+
+            // Displays all bindings grouped by their category.
+            foreach (IGrouping<KeyBindings.KeyActionCategory, KeyValuePair<KeyBindings.KeyAction, KeyBindings.KeyActionDescriptor>> group
```

Comment:
Maybe this is one of those places where `var` would actually be a good idea, since the type takes up the whole line here and can be inferred pretty easily by `AllBindings().GroupBy(binding => binding.Value.Category)`.